packages/api/package.json

```diff
@@ -18,7 +18,7 @@
   },
   "dependencies": {
     "@anatine/zod-nestjs": "^1.10.0",
-    "@clerk/clerk-sdk-node": "4.10.5",
+    "@clerk/clerk-sdk-node": "^4.12.1",
```

Comment:
I remember we had to downgrade the clerk version to make everything work with clerk. Can you not test this locally?

also, check the changelog for the latest release of clerk node sdk to see if they speak about bug fixing if you haven’t already

Reply:
I tested locally and everything seems to be working fine but I can't reproduce the bug locally